editor.planx.uk/src/ui/editor/ComponentTagSelect.tsx

```diff
@@ -17,65 +16,55 @@ interface Props {
   onChange: (values: NodeTag[]) => void;
 }
 
-export const ComponentTagSelect: React.FC<Props> = ({ value, onChange }) => {
-  const isPlatformAdmin = useStore().user?.isPlatformAdmin;
-
-  const showPlaceholderTag = (tagName: string) =>
-    tagName.toLowerCase() === "placeholder" && !isPlatformAdmin ? true : false;
-
-  const renderOption: AutocompleteProps<
-    NodeTag,
-    true,
-    true,
-    false,
-    "div"
-  >["renderOption"] = (props, tag, { selected }) => {
-    const tagName = TAG_DISPLAY_VALUES[tag].displayName;
-
-    if (showPlaceholderTag(tagName)) return;
+const renderOption: AutocompleteProps<
+  NodeTag,
+  true,
+  true,
+  false,
+  "div"
+>["renderOption"] = (props, tag, { selected }) => {
+  if (TAG_DISPLAY_VALUES[tag].isEditable) return null;
```

Comment:
This reads as incorrect to me - if it's editable, don't return anything. 

I'd expect the opposite behaviour - if a tag is _not_ editable, don't display it.